packages/edit-site/src/components/actions/index.js

```diff
@@ -54,19 +65,31 @@ export const trashPostAction = {
 						variant="primary"
 						onClick={ async () => {
 							try {
-								await deleteEntityRecord(
-									'postType',
-									post.type,
-									post.id,
-									{},
-									{ throwOnError: true }
+								await Promise.all(
```

Comment:
I think [Promise.allSettled()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled) is a better fit here, because if some item fails, we could try the remaining ones.